core-bundle/src/Resources/contao/config/config.php

```diff
@@ -334,7 +329,11 @@
 		'symlinks' => array
 		(
 			'callback' => array('Automator', 'generateSymlinks')
-		)
+		),
+		'pages' => array
```

Comment:
Why did you move this down here?

Reply:
Just to group the ones that output the number of files and the ones that do not :)